resources/nl-core/nl-core-MedicationAdministration2.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="nl-core-MedicationAdministration2" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicationAdministration2" />
+  <name value="NlcoreMedicationAdministration2" />
+  <title value="nl core MedicationAdministration2" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <purpose value="A derived profile from [zib-MedicationAdministration2](http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAdministration2) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="Medication-Process-v9-2-0-0" />
+    <uri value="https://decor.nictiz.nl/art-decor/decor-datasets--mp-?id=&amp;amp;amp;effectiveDate=&amp;amp;amp;conceptId=&amp;amp;amp;conceptEffectiveDate=" />
+    <name value="Medication Process 9 v2.0.0" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="MedicationAdministration" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAdministration2" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="MedicationAdministration">
+      <path value="MedicationAdministration" />
+      <alias value="nl-core-MedicationAdministration2" />
+    </element>
+    <element id="MedicationAdministration.extension">
+      <path value="MedicationAdministration.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationAdministration.extension:pharmaceuticalTreatmentIdentifier">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="pharmaceuticalTreatmentIdentifier" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PharmaceuticalTreatment.Identifier" />

```

Comment:
Kan extensie niet controleren

Reply:
Staat in de MedicationProfiles branch. Dit was de MedicationTreatment extensie maar daarvan heb ik de naam aangepast zodat het meer in lijn staat met de FO en andere Engelse termen.